tests/unit/s2n_client_finished_test.c

```diff
@@ -61,10 +61,8 @@ int main(int argc, char **argv)
         client_conn->actual_protocol_version = S2N_TLS12;
 
         /* Mutate valid verify_data */
-        POSIX_GUARD(s2n_prf_client_finished(server_conn));
-        server_conn->handshake.client_finished[0]++;
-
         EXPECT_SUCCESS(s2n_client_finished_send(client_conn));
+        client_conn->handshake.io.blob.data[10]++;
```

Comment:
This probably needs a comment. You're mutating a part of the handshake message you know is the verify_data? How do you know "10" is right?